rakelib/preferences.rake

```diff
@@ -0,0 +1,88 @@
+namespace :preferences do
+  desc 'Seed the database with the MVP Preference and associated PreferenceChoices'
+  task initial_seed: :environment do
+    tracking = {
+      starting_preference_count: Preference.count,
+      starting_preference_choice_count: PreferenceChoice.count,
+      final_preference_count: 0,
+      final_preference_choice_count: 0,
+      errors: 0
+    }
+
+    begin
+      preference = Preference.find_or_create_by!(code: preference_attrs[:code]) do |pref|
+        pref.title = preference_attrs[:title]
+      end
+      tracking[:final_preference_count] += 1
+
+      preference_choice_attrs.each do |attrs|
+        PreferenceChoice.find_or_create_by!(code: attrs[:code]) do |pref_choice|
+          pref_choice.description   = attrs[:description]
+          pref_choice.preference_id = preference.id
+        end
+
+        tracking[:final_preference_choice_count] += 1
+      end
+
+      p tracking
+    rescue StandardError => e
+      tracking[:errors] += 1
+      message = "While initially seeding Preferences, experienced this error: #{e}"
+
+      p message
+      Rails.logger.error message
+    end
+  end
+end
+
+def preference_attrs
+  {
+    code: 'benefits',
+    title: 'the benefits a veteran is interested in, so VA.gov can help you apply for them'
+  }
+end
+
+def preference_choice_attrs
```

Comment:
I elected to keep the seed data as a method in the rake file, as opposed to creating a yaml file, etc.  Reduces the complexity for this one-off production seeding.